test/cppsuite/configs/search_near_01_stress.txt

```diff
@@ -13,10 +13,14 @@ workload_generator=
     (
         collection_count=3,
         key_count_per_collection=100,
-        key_size=50,
+        key_size=5,
     ),
     read_config=
     (
+        # The key size parameter is used to generate the prefix 
+        # used in search_near. Use a smaller size to increase
+        # the chance to match an existing key.
+        key_size=3,
```

Comment:
@jiechenbo , would I be wrong if I say the `key_size` is not used in `search_near_01.cxx`?

Reply:
@jiechenbo Is the key_size needed here?

Reply:
Yep! thats correct, key size is not utilised in search near 01. For each run, we randomise the search key size between 1 - 3, and use that size throughout the test.